tensorflow_datasets/core/dataset_info_test.py

```diff
@@ -301,6 +302,13 @@ def test_updates_on_bucket_info(self):
     # These are dynamically computed, so will be updated.
     self.assertEqual(40, info.splits.total_num_examples)
     self.assertEqual(2, len(info.as_proto.schema.feature))
+    
+  def test_show_examples(self):
+    with testing.tmp_dir(self.get_temp_dir()) as tmp_dir:
+      builder = RandomShapedImageGenerator(data_dir=tmp_dir)
+    builder.download_and_prepare()
+    ds = builder.as_dataset(split='train')
+    #builder.info.show_examples(ds)
```

Comment:
You can use a mock library for your problem. Please check this [link](https://stackoverflow.com/questions/24343640/how-to-run-nosetests-without-showing-of-my-matplotlibs-graph).

Reply:
Ok, thanks for pointing this. I am checking it.

Reply:
+1, mock seems the right tool to use